### PR TITLE
Add WeChat and Alipay logo

### PR DESCRIPTION
**Pull request by tachyons**

### CODE REVIEW

**Comment by tommi-lew** [comment]
As spoken, there are other sprites which has to be updated too.

File: app/assets/stylesheets/sephora_style_guide/variables/_icons.scss [hunk 1]
```diff
@@ -29,13 +29,13 @@ $icon_50x30-top: -1 * $icon_30x20-height;
 $icon_50x30-classes: twoc2p unionpay bca amex mastercard molpay webcash visa
               ambank fpx paypal tmb kkbank krungsri scb card16 uob payatpost
               tesco bbl truemoney tbank bigc mpay jautpay pubbank gcash
-              cstore bni;
+              cstore bni alipay wechat alipay_hk;
```
Comment (tommi-lew) [inline app/assets/stylesheets/sephora_style_guide/variables/_icons.scss]: @tachyons How about `alipay_cn` and `alipay_hk` for consistency ?

Comment (tommi-lew) [reply app/assets/stylesheets/sephora_style_guide/variables/_icons.scss]: Also forgot, `wechatpay_cn`. In case WeChat Pay HK is introduced at some point.

Comment (tachyons) [reply app/assets/stylesheets/sephora_style_guide/variables/_icons.scss]: I thought about the same, but adyen calls alipay china  as alipay and Alipay Hongkong as alipay_hk. I thought about using same naming convention

Comment (tommi-lew) [reply app/assets/stylesheets/sephora_style_guide/variables/_icons.scss]: @tachyons Oh wait, is Adyen's naming convention tied to our naming convention here?

Comment (tachyons) [reply app/assets/stylesheets/sephora_style_guide/variables/_icons.scss]: No, there is no hard coupling, just thought about following the same. WDYT ?

**Pull request merged**